Bindings/Java/Matlab/configureOpenSim.m.in

```diff
@@ -39,7 +39,7 @@
             error('User cancelled function')
         end
         % Check if the correct OpenSim directory is selected
-        if ~exist(fullfile(openSimFolder, '@CMAKE_INSTALL_SYSCONFDIR@', 'buildinfo.txt'))
+        if ~exist(fullfile(openSimFolder, '@CMAKE_INSTALL_SYSCONFDIR@', 'OpenSim_buildinfo.txt'))
```

Comment:
Do you expect multiple buildinfo.txt files one per package? Wouldn't it be clear from the context/folder name?